core/src/test/scala/scalacache/PackageObjectSpec.scala

```diff
@@ -20,25 +20,25 @@ class PackageObjectSpec extends FlatSpec with Matchers with BeforeAndAfter with
   behavior of "#get"
 
   it should "call get on the cache found in the ScalaCache" in {
-    scalacache.get("foo")
+    scalacache.get[String]("foo")
```

Comment:
Does adding the `: Codec` cause type inference to stop working as expected? 

e.g. if this was `val foo: String = scalacache.get("foo")` would it still compile without a type annotation? If not, that's kind of a shame, but not the end of the world.

Somebody somewhere probably has some code that will need a type annotation to be added, so we should put a warning in the changelog.